### ISSUE TITLE
[Question] Website embedding in yetiforce not working

### ISSUE THREAD

**Issue by rohanjhanepal** [YetiForceCompany/YetiForceCRM#13549]
### Description

I am trying to embed a website in My site section and when i click on the website i get this issue:

The website that you are trying to open is not secure and might not open. If you still wish to view the webpage then you can click on the content blocker in the address bar and enable it.

What do i do?

![Screenshot (36)](https://user-images.githubusercontent.com/40380541/94284446-16accf00-ff72-11ea-9246-b3f307d58bae.png)

**Comment by mariuszkrzaczkowski** [comment]
![github](https://user-images.githubusercontent.com/10198654/94396107-7e753c80-0161-11eb-9409-839abc5e375c.jpg)

**Comment by rioshaz** [comment]
@rohanjhanepal please post proper bug report.. system environment, server, yf version, log etc.. dont take shortcuts...

**Comment by rskrzypczak** [comment]
The module you mentioned is very old, we inherited it from Vtiger. It would have to be updated, however I guess it would be quicker if we wrote it from scratch.
For the module to work we would have to get rid of a few restrictions, which could cause some  vulnerabilities.
I think the fact that people aren't very interested in it, and it has low quality code impacted the decision to remove it. 
So it won't be present in the new version.
